### PR TITLE
Updated MLO

### PR DESCRIPTION
**Pull request by ineiti**

### CODE REVIEW

**Review by cgrigis — CHANGES_REQUESTED** [review]

File: data/MLO/projects.yaml [hunk 1]
```diff
@@ -80,7 +84,11 @@ projects:
     information:
       - type: Paper
         title: "COLA: Decentralized Linear Learning"
-        url: https://arxiv.org/abs/1808.04883
+        url: https://infoscience.epfl.ch/record/266638
```
[the remark this answers is not in the document]
Comment (ineiti) [reply data/MLO/projects.yaml:87]: Oups - no. Then let's keep the arxiv one...

File: data/MLO/projects.yaml [hunk 2]
```diff
@@ -182,12 +187,16 @@ projects:
     information:
       - type: Paper
         title: "PowerSGD: Practical Low-Rank Gradient Compression for Distributed Optimization"
-        url: https://arxiv.org/abs/1905.13727
+        url: https://infoscience.epfl.ch/record/278542
```
[the remark this answers is not in the document]
Comment (ineiti) [reply data/MLO/projects.yaml:190]: It is available here: `Additional link: fulltext` -> `Paper` - but it's cumbersome and not the `Download` on top that the other papers have.

I wrote to Martin to ask whether this is on purpose, or if he wants to add the PDF there.

File: data/MLO/projects.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ineiti) [reply data/MLO/projects.yaml:87]: Went back to infoscience, now that martin added the pdf there.

File: data/MLO/projects.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ineiti) [reply data/MLO/projects.yaml:190]: OK, PDF is added to infoscience.

**Review by cgrigis — APPROVED** [review]

**Pull request merged**